src/wilcoxon.jl

```diff
@@ -163,6 +163,27 @@ function StatsAPI.pvalue(x::ExactSignedRankTest; tail=:both)
     end
 end
 
+function psignrank(W, n, lower_tail)
```

Comment:
Might be good to add a docstring and constrain the arguments?

```suggestion
function psignrank(W::T, n::T, lower_tail::Bool) where {T<:Integer}
```
(promotion could be done separately if necessary)

Reply:
Given that the test only allows for `n::Int` maybe at least initially we could just support `W::Int` and `n::Int`.

Reply:
I agree that W only makes sense as integer in the exact test with no ties. However, the caller of `psignrank` seems to provide a float, I assume to accommodate for approximate test/ties. 

We could just round the float to the nearest integer before we call `psignrank`.